openpype/settings/entities/schemas/projects_schema/schema_project_slack.json

```diff
@@ -0,0 +1,89 @@
+{
+    "type": "dict",
+    "key": "slack",
+    "label": "Slack",
+    "collapsible": true,
+    "is_file": true,
+    "children": [
+
+        {
+            "type": "dict",
+            "collapsible": true,
+            "key": "publish",
+            "label": "Publish plugins",
+            "children": [
+                {
+                    "type": "dict",
+                    "key": "CollectSlackFamilies",
+                    "label": "Notification to Slack",
+                    "use_label_wrap": true,
+                    "children": [
+                        {
+                            "type": "label",
+                            "label": ""
+                        },
+                        {
+                            "type": "boolean",
+                            "key": "enabled",
+                            "label": "Enabled"
+                        },
+                        {
+                            "type": "boolean",
+                            "key": "optional",
+                            "label": "Optional"
+                        },
+                        {
+                            "type": "text",
```

Comment:
Should the token be stored in collector settings? If you want to access it outside publish context you have to know that it's stored in collector plugin settings.

Reply:
Moved it outside publish plugins